Lydian.Unity.Automapper/Core/UnityCollectionFacade.cs

```diff
@@ -4,25 +4,25 @@
 
 namespace Lydian.Unity.Automapper.Core
 {
-	/// <summary>
-	/// A facade on top of a Unity call to ResolveAll for a particular type.
-	/// </summary>
-	/// <typeparam name="T"></typeparam>
-	internal sealed class UnityCollectionFacade<T> : IEnumerable<T>
+    /// <summary>
+    /// A facade on top of a Unity call to ResolveAll for a particular type.
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    internal sealed class UnityCollectionFacade<T> : IEnumerable<T>
 	{
-		private readonly IEnumerable<T> resolvedTypes;
-
-		/// <summary>
+	    private readonly IUnityContainer _container;
+ 
+	    /// <summary>
 		/// Initializes a new instance of the CollectionFacade class.
 		/// </summary>
 		public UnityCollectionFacade(IUnityContainer target)
 		{
-			resolvedTypes = target.ResolveAll<T>();
```

Comment:
Hmmm. Slight change in behaviour here - is there a reason behind this? I suspect what you're proposing is to get the latest version all the time?